### DISCUSSION TITLE
Webhook in Elsa

### DISCUSSION THREAD

**Discussion by mubukp** [elsa-workflows/elsa-core#2718] [Q&A]
Do you have any specific samples on webhook usage in Elsa?

**Comment by sfmskywalker** [comment]
Not yet, but would be good to add. @jwillmer perhaps a good follow-up item for someone from the team?

**Reply by jwillmer** [reply]
Webhooks are just HTTP endpoints. Currently it is not very intuitive how to store and continue processing received data but that is a general issue and not unique to the HTTP activity. How to setup the HTTP activity in order to receive events pretty much depends on the webhook type. Usually it is just a GET endpoint to a specific URL.

In short:
- set a common name for the activity to reference it in other activities and via intellisense
- add the JSON schema you hope to receive in the HTTP endpoint in order to work with intellisense in other activities
